pandas/plotting/_matplotlib/hist.py

```diff
@@ -2,6 +2,7 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
+from scipy.stats import gaussian_kde
```

Comment:
The `scipy` ones should be moved. It's the same case as with matplotlib, we don't require it unless you call a function that uses it. In this case, if you call a plot (like the line plot), you'll need matplotlib, but not scipy, which you only need if you call the kde plot.

This is why the tests are failing (not sure if you've seen the CI is in red). Can you restore the scipy ones please?

Reply:
Got it! Thanks!